src/Amadevus.RecordGenerator.Generators/SyntaxSnippetGenerators.cs

```diff
@@ -0,0 +1,155 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Amadevus.RecordGenerator.Generators
+{
```

Comment:
I don't like those helper classes. Soon we'll be able to have dependencies in generators and I'd then use SyntaxGenerator probably, but until then let's keep `SyntaxExtensions` for API touchups and not build those big helpers.

Reply:
Removed helper classes, merged equality generation into single class